### PR TITLE
display version in cli

### PR DESCRIPTION
**Pull request by TvrtkoM**
This adds version number to cli output (#57)

### CODE REVIEW

File: src/lib/fs/fsUtils.ts [hunk 1]
```diff
@@ -51,6 +68,24 @@ export function readFile(path: string): string {
...
+/**
+ * Reads the package.json file at path and returns javascript object.
+ * If path is not given, first package.json found in one of parent directories is used.
+ *
+ * @param path Optional path to package.json file
+ * @returns [[PackageJsonFile]] object
+ * @throws if path does not point to a file
+ */
+export function readPackageJson(path?: string): PackageJsonFile {
+  if (path == null) {
```
Comment (warriordog) [inline src/lib/fs/fsUtils.ts:80]: Could you make this === undefined? ESLint is set to require === instead of == but I guess it didn't run?

**Review by warriordog — CHANGES_REQUESTED** [review]
Thank you for this contribution! Everything looks good, but somehow a double-equals comparison slipped in. I'm surprised ESlint didn't catch it, since the eqeqeq rule is enabled. If you can change that to a triple-equals comparison then I'll merge this PR.

File: src/lib/fs/fsUtils.ts (hunk 1 above)
Comment (TvrtkoM) [reply src/lib/fs/fsUtils.ts:80]: ESLint is allowing comparing with null using double-equals (allow-null-check option is set to true)(https://github.com/palantir/tslint/blob/master/src/configs/recommended.ts#L85)

Comment (warriordog) [reply src/lib/fs/fsUtils.ts:80]: Oh I forgot I set that, thanks for reminding me. I might remove that setting later but for now its fine.

**Review by warriordog — APPROVED** [review]

**Pull request merged**